### DISCUSSION TITLE
Write access request

### DISCUSSION THREAD

**Discussion by ahmetsait** [desjarlais/Scintilla.NET#137] [General]
Hi @desjarlais,
I tried to send an email first but in case you didn't receive it I wanted to ask it here as well.
Would you like to add me as a collaborator to the ScintillaNET repository? I might be able to take some of the responsibility off your shoulders.

Thanks.

**Comment by desjarlais** [comment]
Absolutely, any help is much appreciated.

**Comment by desjarlais** [comment]
@ahmetsait you have been added as a collaborator.  Thank you for being willing to help and for everything you've done so far, it is really quite impressive.
